.github/workflows/_linux-build.yml

```diff
@@ -139,10 +139,6 @@ jobs:
           pip3 install requests==2.26 boto3==1.16.34
           python3 -m tools.stats.upload_binary_size_to_scuba || exit 0
 
-      - name: Chown workspace
-        uses: ./.github/actions/chown-workspace
```

Comment:
if this works, we should remove the .github/actions/chown-workspace too

Reply:
There are few more cases where it is still used (for example for nightly pipelines)
But eventually, yes, we should get rid of it